cypress/locators/pager/index.js

```diff
@@ -21,6 +22,7 @@ export const pageSelect = () =>
 export const pageSelectElement = () => cy.get(PAGE_SELECT);
 export const maxPages = () => cy.get(MAX_PAGES);
 export const currentPageWrapper = () => cy.get(CURRENT_PAGE);
+export const currentPageWLabelWrapper = () => cy.get(CURRENT_PAGE_LABEL);
```

Comment:
```suggestion
export const currentPageLabelWrapper = () => cy.get(CURRENT_PAGE_LABEL);
```

Reply:
You'll also need to change the name if you've used it somewhere else too.

Reply:
Classic me